src/content/documentation/publish/add-on-policies-preview-2025-08.md

```diff
@@ -172,13 +185,13 @@ If the add-on uses native messaging, the Add-on Policies (including those relate
 
 ### User Consent and Control
 
-The user must be provided with a clear way to control the add-on’s data transmission, either through a consent experience created by the add-on developer, or by using Firefox’s built in data collection and transmission consent experience.
+The user must be provided with a clear way to control the add-on’s data transmission, either through a consent experience created by the add-on developer, or by using Firefox’s built in data collection and transmission consent experience. In the case of add-ons that qualify for implicit consent, under the “Implicit Consent for Self‑Evident, Single‑Use Data Transmission” policy, installation is the consent.
```

Comment:
Replace U+02011 non-breaking hyphen (`‑`) with U+002 hyphen-minus (`-`).

While these characters are essentially identical from a visual point of view, they have slightly different text wrapping behaviors and are spoken differently by some by some text to speech engines. 

```suggestion
The user must be provided with a clear way to control the add-on’s data transmission, either through a consent experience created by the add-on developer, or by using Firefox’s built in data collection and transmission consent experience. In the case of add-ons that qualify for implicit consent, under the “Implicit Consent for Self-Evident, Single-Use Data Transmission” policy, installation is the consent.
```